CMakeLists.txt

```diff
@@ -15,3 +15,9 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
   endif()
 endif()
 
+install(
+  DIRECTORY                include/EngFormat-Cpp
+  DESTINATION              include
+  CONFIGURATIONS           ${build_type_list}
```

Comment:
This is VT specific.

Reply:
Removed and changed CI to test `install` target as well.